server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v1/SqlDiagnoseController.java

```diff
@@ -75,4 +79,12 @@ public SuccessResponse<TraceSpan> getFullLinkTrace(@PathVariable String sid, @Re
         return Responses
                 .success(diagnoseService.getFullLinkTrace(sessionService.nullSafeGet(SidUtils.getSessionId(sid)), sql));
     }
+
+    @ApiOperation(value = "getFullLinkTraceJson")
+    @RequestMapping(value = "/getFullLinkTraceJson/{sid}", method = RequestMethod.POST)
```

Comment:
what is the difference between getFullLinkTrace and getFullLinkTraceJson ?
the  previous one return json format also.

Reply:
This api is a new interface to query json content, only for download json file. It is not the same api as the previous one

Reply:
however we cannot distinguish from the API endpoint. 

if for download, here we may return a file id then front-end call file download API
- step1, getFulLinkTraceDownloadLink, return download link
- step, call download link, which is a cloud storage url in cloud env

if for query jager compatible, may share the same API wieh getFullLinkTraceJson, add an optional queryparameter `jagerCompatible=true`, then download logic is controlled by front-end

Reply:
changed this api into `/getFullLinkTraceDownloadUrl` which would return a download link ~